CONFIGURING-DETECTORS.md

```diff
@@ -0,0 +1,103 @@
+
+### `edge_inference_configs`
+
+Edge inference configs are 'templates' that define the behavior of a detector on the edge. Each detector you configure will be assigned one of these templates. There are some pre-defined configs that represent the main ways you might want to configure a detector. However, you can edit these and also create your own as you wish.
+
+#### Structure of an edge_inference_config
+
+For each edge inference config, you can configure various parameters. For a complete description of each, see [Appendix: Edge Inference Parameters](#appendix-edge-inference-parameters) below.
+
+#### Pre-defined edge inference configs
+
+There are a few pre-defined configs that represent the main ways you might want to configure a detector. 
+
+##### `default`
+```
+default: # Return the edge model's prediction if sufficiently confident; otherwise, escalate to the cloud.
+    enabled: true
+    always_return_edge_prediction: false
+    disable_cloud_escalation: false
+```
+This is the default behavior for a detector on the edge and is likely what you'll want to use most of the time, unless you have a specific reason to use a different configuration.
+
+##### `edge-answers-with-escalation`
+```
+edge-answers-with-escalation: # Always return the edge model's predictions, but still escalate to cloud if unconfident.
+    enabled: true
+    always_return_edge_prediction: true
+    disable_cloud_escalation: false
+    min_time_between_escalations: 2.0
+```
+Use this config if: you want the fastest answers from the edge model (regardless of confidence), but also want unconfident queries to be escalated to the cloud at a reasonable rate so that you can provide labels and continue improving the models.
+
+##### `no-cloud`
+```
+no-cloud: # Always return the edge model's prediction and never escalate to the cloud.
+    enabled: true
+    always_return_edge_prediction: true
+    disable_cloud_escalation: true
+```
+Use this config if: you always want the fastest answers from the edge model and don't want any image queries to get escalated to the cloud.
+
+##### `disabled`
+```
+disabled:
+    enabled: false
+```
+Use this config if: you don't want the edge endpoint to accept image queries for this detector.
+
+### `detectors`
+
+This section is where you define your detectors, along with the edge inference config to use for each of them. The structure looks like:
+```
+detectors:
+    - detector_id: "det_abc"
+        edge_inference_config: "default"
+    - detector_id: "det_xyz"
+        edge_inference_config: "no-cloud"
+```
+Add a new entry for each detector that you want to configure. Remember that inference configs can be applied to as many detectors as you'd like, so if you want multiple detectors to have the same configuration, just assign them the same edge inference config. Make sure you don't have multiple entries for the same detector - in this case, the edge endpoint will error when starting up.
+
+## Appendix: Edge Inference Parameters
```

Comment:
I'm confused why this is a separate section

Reply:
Nevermind I get it. I missed this part:
```There are a few pre-defined configs that represent the main ways you might want to configure a detector. For a complete description of each, see [Appendix: Edge Inference Parameters](#appendix-edge-inference-parameters) below. ```

Can we name it something besides appendix though? Because as a reader I'm inclined to ignore appendices.